### DISCUSSION TITLE
Unignore über die UI (>=1.65.0)

### DISCUSSION THREAD

**Discussion by SukramJ** [SukramJ/aiohomematic#1647] [General]
[Unignore](https://github.com/danielperna84/hahomematic/blob/devel/docs/unignore.md) kann jetzt auch über die [UI](https://github.com/danielperna84/hahomematic/blob/devel/docs/unignore.md#using-the-ui) konfiguriert werde.

**Comment by SukramJ** [comment]
Moin @Baxxy13,
hast Du als mein unignore Experte mal wieder Lust was Neues zu Testen?
aber der Version [1.65.0b2 ](https://github.com/danielperna84/custom_homematic/releases/tag/1.65.0b2) lässt sich unignore über die UI konfigurieren.
Die Datei bleibt erst mal erhalten, und wird zusätzlich herangezogen.

**Reply by Baxxy13** [reply]
Hi, bin gerade aus dem Urlaub zurück.
Ich guck's mir dann die nächsten Tage an.

**Comment by Baxxy13** [comment]
So, ich habe jetzt mal reingeschaut.
Das hast du ziemlich cool (und versteckt) umgesetzt. 
Mit den 5-6 getesteten (mir gebräuchlichen) Parametern konnte ich erstmal keine Probleme feststellen. 

Ich war erstaunt wie viele Parameter da selbst bei meinem kleinen angebundenen Testsystem auftauchen. 
Grundsätzlich bin ich ja eher für "lieber mehr als weniger", aber die Liste ist so enorm lang das mein erster Gedanke war... "da sollten wir was wegfiltern". 

Andererseits ist das ja ein Feature für Nerds (wie mich) und User die "genau diesen einen Parameter" über HA ändern möchten. Da wäre eine Filterung kontraproduktiv.

Nach etwas hin und her überlegen würde ich sagen. "Gib uns alles", also keine Vorfilterung. Wer damit hantieren will sollte schon wissen was er macht. 😉 

Und jetzt wo wir "drüber reden" muss ich mich erstmal entschuldigen das ich es bisher nicht geschafft habe den Wiki-Eintrag zum Thema zu verfassen.  Steht auf meiner Agenda, kommt... irgendwann.

**Reply by SukramJ** [reply]
Danke fürs Testen.
Die "Filter" sind read & event bei `VALUES` und read bei `MASTER`.

> Und jetzt wo wir "drüber reden" muss ich mich erstmal entschuldigen das ich es bisher nicht geschafft habe den Wiki-Eintrag zum Thema zu verfassen. Steht auf meiner Agenda, kommt... irgendwann.

Kein Problem.

**Comment by pduck27** [comment]
Hallo, irgendwie bin ich zu deppert. Ich würde gerne den VALVE_STATE mit auf den hmip_etrv_2 - Entitäten haben. Da bei zu geringer Batterie die Dinger letztlich bei der Entkalkungspfad im ADAPT hängen bleiben wäre das mein Notfallskript zur Meldung leerer Batterien. Aber ich finde auf der Konfiguration-Page der Integration nirgends den Punkt "advanced settings" um mit den unignore zu spielen und den VALVE_STATE einzubinden. Laut PARAM_SETS ist er ja da.
Mein HA läuft auf einem Proxmox als Docker mit den helper-scripts: https://tteck.github.io/Proxmox/#home-assistant . Ist da das Problem?

**Reply by SukramJ** [reply]
Schade, leider erfüllst du damit nicht die Zulassungsbedingungen für Unignore.
Du must also selbstständig den bereits vorhandenen aber deaktivierten Sensor Ventil-Öffnungsgrad beim HmIP-eTRV-2 aktivieren.

**Reply by Baxxy13** [reply]
Disqualifiziert? Warum?
Er/sie möchte doch VALVE_STATE und nicht LEVEL.

Könnte ich bei mir auswählen:
![Screenshot 2024-10-30 180519](https://github.com/user-attachments/assets/8d083cef-f0a8-435b-ba20-9de8fa1b19d4)

**Reply by SukramJ** [reply]
Ich urteile nur hart, und schau nicht nach.

**Reply by pduck27** [reply]
Danke, der Screenshot hilft mir wie es aussehen muss und dass das mit VALVE_STATE geht. Allerdings ist mein Problem woanders, HACS hat ein zu alte Version im Repo (1.58). Deswegen habe ich den Punkt nirgends gefunden.

**Reply by SukramJ** [reply]
Und HA und HACS sind auch aktuell?

**Reply by pduck27** [reply]
Hallo, jetzt passt alles nach einer Updaterunde. Vielen Dank für die Hilfe, das UNIGNORE Feature läuft wunderbar :)
